docs/practitioners/retrieveBOandL.md

```diff
@@ -1,7 +1,7 @@
 # Retrieve Board Orders and Licenses
 
-Retrieves Board Orders and Licenses associated with a federation ID
- 
+Retrieves Board Orders and Licenses associated with a FID
```

Comment:
Shouldn't be capitalizing non-starter words (e.g. Board Orders). This is inconsistent with other examples.